src/components/therapy/TreatmentPlanManager.tsx

```
@@ -0,0 +1,651 @@
+  };
+
+  const handleObjectiveChange = (
+    goalIndex: number, 
+    objIndex: number, 
+    field: keyof ClientSideNewObjective | keyof EditableObjective, 
+    value: string, 
+    isEdit = false
+  ) => {
+    if (isEdit && editingPlanData) {
+      const updatedGoals = JSON.parse(JSON.stringify(editingPlanData.goals || [])) as EditableGoal[];
+      if (updatedGoals[goalIndex] && updatedGoals[goalIndex].objectives[objIndex]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)